scripts/klippy-requirements.txt

```diff
@@ -9,3 +9,4 @@ greenlet==3.0.3 ; python_version >= '3.12'
 Jinja2==2.11.3
 python-can==3.3.4
 markupsafe==1.1.1
+sensirion-gas-index-algorithm==3.2.1
```

Comment:
this library requires at least python 3.6[^1], but klipper is still testing against python 2. I'm sure this is intentional, but I'm not sure why. It went end of life years ago[^2].

[^1]: https://pypi.org/project/sensirion-gas-index-algorithm/
[^2]: https://www.python.org/doc/sunset-python-2/

Reply:
Thx for your feedback!

That was also a reason why I opened the post in the Klipper discourse, "if I should use a python dependency" or upload the algorithm itself in a file in the Klipper repo.

I didn't continue to develop on this PR until I get an answer for this question...

Reply:
I think this test failure answers the question for you, lol. To make this python2 compatible, another implementation of the algorithm is needed. Putting it in klipper itself seems like the most stable option to me.

Reply:
There are also python 2 compatible libs available. That's why I ask... I hope I have time this weekend to continue here. Thx for your feedback.